lib/Optimizer/Transforms/GenKernelExecution.cpp

```diff
@@ -834,6 +834,9 @@ class GenerateKernelExecution
     if (failed(irBuilder.loadIntrinsic(module, cudaq::llvmMemCopyIntrinsic)))
       return module.emitError(std::string("could not load ") +
                               cudaq::llvmMemCopyIntrinsic);
+    if (failed(irBuilder.loadIntrinsic(module, cudaq::llvmMemSetIntrinsic)))
+      return module.emitError(std::string("could not load ") +
+                              cudaq::llvmMemSetIntrinsic);
```

Comment:
This error is "impossible" if the code is correctly built. Ignore the codecov complaint.